lib/dialyzer/src/dialyzer_callgraph.erl

```diff
@@ -337,15 +337,15 @@ module_postorder_from_funs(Funs, #callgraph{digraph = DG,
   digraph_delete(SubGraph),
   {PO, CG#callgraph{active_digraph = Active}}.
 
+%% We KNOW that `error` is not a valid value in the table.
 ets_lookup_dict(Key, Table) ->
-  try ets:lookup_element(Table, Key, 2) of
-      Val -> {ok, Val}
-  catch
-    _:_ -> error
+  case ets:lookup_element(Table, Key, 2, error) of
+    error -> error;
+    Val -> {ok, Val}
   end.
 
 ets_lookup_set(Key, Table) ->
-  ets:lookup(Table, Key) =/= [].
+  ets:member(Table, Key).
```

Comment:
Why is this (`=/= []`) no longer needed?

Reply:
[ets:lookup/2](https://www.erlang.org/doc/man/ets.html#lookup-2) returns an empty list if the key is not present in the table, and otherwise a list with one or more elements. The original code tests whether the there is an object with that key in the table, which is exactly what `ets:member/2` does.